sdks/java/expansion-service/src/main/java/org/apache/beam/sdk/expansion/service/ExpansionServiceOptions.java

```diff
@@ -58,6 +58,12 @@ public interface ExpansionServiceOptions extends PipelineOptions {
 
   void setExpansionServiceConfig(ExpansionServiceConfig configFile);
 
+  @Description("Starts an Expansion Service with support for Google ALTS authentication")
```

Comment:
Technically AltsServerBuilder or gRPC in general is not limited to google, though in practice it may be pretty much google specific. I am wondering if we could phrase the description like https://github.com/grpc/grpc-java/blob/58bab7434a9134d25d55b95ed986ab9e7d715c6a/alts/src/main/java/io/grpc/alts/AltsServerBuilder.java#L38 so it does not be a google specific thing, as expansion-service itself does not depend on google services